src/main/kotlin/no/nav/familie/ef/iverksett/infrastruktur/transformer/IverksettDtoToDomain.kt

```diff
@@ -67,7 +67,7 @@ fun VedtaksperiodeDto.toDomain(): Vedtaksperiode {
 
 fun VedtaksdetaljerDto.toDomain(): Vedtaksdetaljer {
     return Vedtaksdetaljer(vedtaksresultat = this.resultat,
-                           vedtaksdato = this.vedtaksdato,
+                           vedtakstidspunkt = this.vedtakstidspunkt,
```

Comment:
this.vedtakstidspunkt ?: throw Feil("Vedtakstidspunkt mangler")  - så kan internobjektet være uten `?`